llvm/lib/Transforms/Vectorize/SandboxVectorizer/Passes/PassRegistry.def

```diff
@@ -0,0 +1,20 @@
+//===- PassRegistry.def - Registry of passes --------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file is used as the registry of sub-passes that are part of the
+// SandboxVectorizer pass.
+//
+//===----------------------------------------------------------------------===//
+
+// NOTE: NO INCLUDE GUARD DESIRED!
+
+#ifndef REGION_PASS
+#define REGION_PASS(NAME, CREATE_PASS)
+#endif
+
+REGION_PASS("null", NullPass())
```

Comment:
Also why use `NullPass()` and not just the class name `NullPass` in `REGION_PASS("null", NullPass())`. Isn't the class name easier to work with?

Reply:
I just followed the pattern for the existing `FUNCTION_PASS` in llvm/lib/Passes/PassRegistry.def. I have no preference one way or the other.